kingpin/actors/group.py

```diff
@@ -50,6 +51,7 @@ class BaseGroupActor(base.BaseActor):
 
     all_options = {
         'contexts': (list, [], "List of contextual hashes."),
+        'context-file': (str, '', "A JSON file with a list of contexts."),
```

Comment:
So ... I think I'd rather re-use `contexts` and simply have the code check for a `str` or a `list` and behave differently. Thoughts?